### PR TITLE
add requestSecurityCode member to BasicCardRequest

### PR DESCRIPTION
**Pull request by ianbjacobs**
Relates to
 https://github.com/w3c/payment-request/issues/97

It does not close that issue because this solution is payment method specific (and also does not address all the fields within Basic Card; however it does address the one that has been cited).

The following tasks have been completed:

 * [X] Confirmed there are no ReSpec errors/warnings.
 * [ ] Added Web platform tests (link)
 * [ ] added MDN Docs (link)

Implementation commitments:

 * [ ] Chrome (link to issue)
 * [ ] [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1546859)
 * [ ] Edge (public signal)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/pull/78.html" title="Last updated on Feb 12, 2020, 4:18 AM UTC (9748e1d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/78/bafe19b...9748e1d.html" title="Last updated on Feb 12, 2020, 4:18 AM UTC (9748e1d)">Diff</a>

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
I'm ok with this

**Comment by marcoscaceres** [comment]
Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1546859

**Comment by marcoscaceres** [comment]
Sent a patch to Gecko. Note that it only patches the DOM implementation. We don't have a basic card UI.

**Review by rsolomakhin — APPROVED** [review]
This looks right :+1:

**Comment by marcoscaceres** [comment]
We should probably add a manual test before merging.

**Comment by rsolomakhin** [comment]
Good idea.

--Rouslan

**Comment by marcoscaceres** [comment]
@rsolomakhin or @danyao, just wondering if this ever landed in Chrome?

**Comment by rsolomakhin** [comment]
This did not land in Chrome yet, @marcoscaceres .

**Comment by marcoscaceres** [comment]
@rsolomakhin thanks for confirming. I guess we should wait till it's in Chrome before we land this, otherwise it might confuse people.

**Comment by Krystosterone** [comment]
I like this and believe this makes sense

**Comment by marcoscaceres** [comment]
@rsolomakhin or @danyao, any update on the Chrome side about supporting this?

**Comment by rsolomakhin** [comment]
Hi @marcoscaceres , at this point we're investing heavily into payment handlers (both web based and OS-specific apps), so let's bypass waiting for Chrome to update it's basic-card implementation. Instead, let's update the spec and encourage payment apps that implement the basic-card standard to implement support for this new field as well. WDYT?

**Comment by marcoscaceres** [comment]
Ah, just stumbled onto https://blog.chromium.org/2020/01/rethinking-payment-request-for-ios.html

Sounds like we might want to wrap up Basic Card entirely then.

**Comment by rsolomakhin** [comment]
> Sounds like we might want to wrap up Basic Card entirely then.

The Basic Card standard can exist for 3rd party payment handlers to implement and for merchants to use, if they wish. It's just not Google's cup of tea to implement it, though. We'd much rather focus on Payment Request and Payment Handler standards.

Edit: More precise wording.

**Comment by ianbjacobs** [comment]
I think that we should maintain Basic Card as a Note.

**Comment by marcoscaceres** [comment]
Ok, the above sounds like a plan.

**Pull request merged**
